src/036/README.md

```diff
@@ -0,0 +1,5 @@
+https://projecteuler.net/problem=036
+
+The decimal number, 585 = 10010010012 (binary), is palindromic in both bases.
```

Comment:
Can you change this to `1001001001<sub>2</sub>`? So it looks like this: 1001001001<sub>2</sub>